### ISSUE TITLE
No rules are processed in GitlabCI

### ISSUE THREAD

**Issue by bernardhalas** [accurics/terrascan#925]
* terrascan version: 1.8.0
* Operating System: Ubuntu 20.04

### Description

Terrascan doesn't process any rules when running via Gitlab's CI. Local processing against the same repo works well.

### What I Did

Running locally works fine (my terraform files are in the `./terraform` directory):
```
docker run -v terraform:/terraform --rm -it --entrypoint /bin/sh accurics/terrascan:1.8.0 -c "/go/bin/terrascan scan -i terraform -d /terraform"
...
Policies Validated  :   757
...
```

However from the Gitlab's CI I get:
```
...
Policies Validated  :	0
```

My `.gitlab-ci.yaml` configuration follows:
```
terrascan:
  stage: linting
  image:
    name: accurics/terrascan:1.8.0
    entrypoint: ["/bin/sh", "-c"]
  script:
  - pwd
  - ls -al
  - ls -al ./terraform
  - /go/bin/terrascan scan -i terraform ./terraform
  - /go/bin/terrascan scan -i terraform -d ./terraform
  - /go/bin/terrascan scan -i terraform terraform
  - /go/bin/terrascan scan -i terraform -d terraform
```
As you can see, I've added extra comands above (`pwd`, `ls`) to make sure I'm in the correct working dir. And I also executed the `terrascan scan` command against various interpretations of the same destination path. All showing the same result: 0 policies validated.

**Comment by kanchwala-yusuf** [comment]
Hi @bernardhalas ,
Thank you for using terrascan!

I have a quick question:
How is the volume being attached to the terrascan container at `./terraform` directory? I cannot see a section in `.gitlab-ci.yaml` shared above. Just wanted to make sure the code is being added into the `terrascan` container, just the way it is being attached with `-v` option when you are running it locally.

**Comment by bernardhalas** [comment]
Thank you for a quick response @kanchwala-yusuf 

Thanks to your input I found out that I wrongly mounted the folder in the `docker run` command. I've fixed this and updated the intial report. This doesn't explain the difference though.

`docker run -v ${PWD}:/project --rm -it --entrypoint /bin/sh accurics/terrascan:1.8.0 -c "/go/bin/terrascan scan -i terraform /project/terraform`

I assume the reason why there were 0 rules processed is because OCI cloud provider isn't supported and hence no rules could be applied. Therefore I'm closing this issue.
